### PR TITLE
Feature/db lists

### PR DESCRIPTION
**Pull request by lliam-mcbean**
compare changes and make sure nothing conflicts

### CODE REVIEW

File: server/db/seeds/seeds.sql [hunk 1]
```diff
@@ -37,14 +37,14 @@ VALUES (3, 23, 'very interesting message');
 
 -- Seeds for products table
 
-INSERT INTO products (lat, long, co2_data)
-VALUES (49.1, 23.5, 103);
+INSERT INTO products (title, upc, lat, long, co2_data)
+VALUES ('beans', 2345, 49.1, 23.5, 103);
```
Comment (geecrypt) [inline server/db/seeds/seeds.sql:41]: same comment here about upc

File: server/routes/index.js [hunk 2]
```diff
@@ -1,5 +1,7 @@
 const searchRoute = require('./search');
+const newList = require('./new-list');
```
Comment (geecrypt) [inline server/routes/index.js:2]: should be consistent with route import naming, i.e. `newListRoute`

File: server/routes/search.js [hunk 3]
```diff
@@ -5,16 +5,37 @@ const router = express.Router();
...
-const searchHelper = (productName) => axios.get(`https://api.spoonacular.com/food/products/search?query=${productName}&apiKey=${process.env.SPOONACULAR_API_KEY}&includeNutrition=true`)
+const searchHelper = (productName) => axios.get(`https://api.spoonacular.com/food/products/search?query=${productName}&apiKey=${process.env.SPOONACULAR_API_KEY}&includeNutrition=true&number=3`)
   .then((res) => res.data)
   .catch((error) => error.message);
 
 module.exports = () => {
+  // Initialize router for get /api/search
   router.get('/', (req, res) => {
-    console.log('params', req.query.productName);
+    // console.log('params', req.query.productName);
```
Comment (geecrypt) [inline server/routes/search.js:15]: would be good to remove commented out code

File: server/server.js [hunk 4]
```diff
@@ -20,16 +19,12 @@ app.use(express.json());
...
-    message: 'Seems to work!',
-    data: data.rows,
-  }));
-});
+// Bring in External Routes
+const { searchRoute, newList } = require('./routes/index');
 
+// External Routes
 app.use('/api/search', searchRoute());
+app.use('/api/lists/new', newList());
```
Comment (geecrypt) [inline server/server.js:27]: Probably going to want to refactor the list routes file to accept multiple CRUD operations like new and edit. So perhaps the route could just be `/api/lists` and the `newList` could contain `router.put('/new',...)` ?

File: src/components/Lists/NewList.jsx [hunk 5]
```diff
@@ -1,22 +1,27 @@
...
 
 export default function ListProducts(props) {
-  const { newProduct } = props;
-  const { title } = newProduct;
-  console.log('newProduct', newProduct);
+  const { list } = props;
+  const { title } = list;
+
+  const submitList = (item) => {
+    axios.put('/api/lists/new', { item }).then(() => console.log('success'));
```
[the remark this answers is not in the document]
Comment (soupIsTheCurrencyOfTheFuture) [reply src/components/Lists/NewList.jsx:10]: just a placeholder. i had to hold off on the new list logic and focus on getting the right product data back first.

**Review by geecrypt — CHANGES_REQUESTED** [review]
Looks good, just a few changes pls

**Pull request merged**